src/main/kotlin/apply/domain/mission/judgmentItem/JudgmentItem.kt

```diff
@@ -0,0 +1,28 @@
+package apply.domain.mission.judgmentItem
```

Comment:
r: 패키지 이름은 소문자로 작성합니다. 내친김에 `apply.domain.evaluationItem`도 변경해 주세요.

```suggestion
package apply.domain.judgmentitem
```